attributes/default.rb

```diff
@@ -32,28 +32,30 @@
 default['openvpn']['key']['message_digest'] = 'sha256'
 
 # Cookbook attributes
-default['openvpn']['key_dir']         = '/etc/openvpn/keys'
-default['openvpn']['signing_ca_key']  = "#{node['openvpn']['key_dir']}/ca.key"
-default['openvpn']['signing_ca_cert'] = "#{node['openvpn']['key_dir']}/ca.crt"
-default['openvpn']['user_query']      = '*:*'
+default['openvpn']['key_dir']          = '/etc/openvpn/keys'
+default['openvpn']['ca_key_filename']  = 'ca.key'
+default['openvpn']['signing_ca_key']   = "#{node['openvpn']['key_dir']}/#{node['openvpn']['ca_key_filename']}"
+default['openvpn']['ca_cert_filename'] = "ca.crt"
```

Comment:
rubocop will be happy if you use single-quotes for a string that has nothing to parse/replace